modules/microshift-blocking-nodeport-access.adoc

```diff
@@ -6,27 +6,27 @@
 [id="microshift-blocking-nodeport-access_{context}"]
 = Blocking external access to the NodePort service on a specific host interface
 
-OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. 
-The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
+OVN-Kubernetes does not restrict the host interface where a NodePort service can be accessed from outside a {product-title} node. The following procedure explains how to block the NodePort service on a specific host interface and restrict external access.
 
 .Prerequisites
 
-* You need access to the cluster as a user with the cluster-admin role.
+* You must have an account with cluster administration privileges.
```

Comment:
All of the steps of this procedure run on the host. The ones that need escalated privileges use sudo. None of the commands use the cluster, so it doesn't matter what sort of cluster access the user has.